pcdsdevices/rs_powersupply.py

```diff
@@ -0,0 +1,46 @@
+import logging
+
+from ophyd import Component as Cpt
+from ophyd import Device, EpicsSignal, EpicsSignalRO
+
+logger = logging.getLogger(__name__)
+
+
+class RSchannel(Device):
+    """
+    Class to define PVs of all channels of the RSPowerSupply NGP800.
+    prefix: str
```

Comment:
Same as the other suggestion:
```suggestion

    Parameters
    ----------
    prefix: str
```